erpnext/shopping_cart/cart.py

```diff
@@ -42,9 +42,9 @@ def get_cart_quotation(doc=None):
 
 	return {
 		"doc": decorate_quotation_doc(doc),
-		"shipping_addresses": [{"name": address.name, "display": address.display}
+		"shipping_addresses": [{"name": address.name, "title": address.address_title, "display": address.display}
```

Comment:
title isnt used anywhere as far i see. Don't think it is necessary to include it here.

Reply:
@marination it is used in erpnext/templates/includes/cart/address_card.html